### PR TITLE
fix(input): climb dom for pointer-events:none targets

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/dom.ts [hunk 1]
```diff
@@ -155,6 +160,8 @@ export class ElementHandle<T extends Node = Node> extends js.JSHandle<T> {
       result.x += point.x / 4;
       result.y += point.y / 4;
     }
+    result.x = (result.x * 100 | 0) / 100;
```
Comment (dgozman) [inline src/dom.ts:163]: Let's instead round at line 193, to handle `_offsetPoint` case as well.

File: test/click.spec.js [hunk 2]
```diff
@@ -483,6 +483,10 @@ module.exports.describe = function({testRunner, expect, playwright, FFOX, CHROMI
       expect(await page.evaluate(() => window.result)).toBe('Was not clicked');
     });
 
+    it('should climb dom for pointer-events:none targets', async({page, server}) => {
+      await page.goto(server.PREFIX + '/input/pointer-events.html');
```
Comment (dgozman) [inline test/click.spec.js:487]: nit: might as well do setContent for easier reading.

`await page.setContent('<button><label style="pointer-events:none">Click target</label></button>')`

**Review by dgozman — APPROVED** [review]

File: src/dom.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/dom.ts:163]: Done

File: test/click.spec.js (hunk 2 above)
Comment (pavelfeldman) [reply test/click.spec.js]: Done

**Pull request merged**